asdf_astropy/resources/schemas/wcs/highlevelwcswrapper-1.0.0.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/wcs/highlevelwcswrapper-1.0.0"
+
+title: Represents a HighLevelWCSWrapper object
+
+description: >-
+    Wrapper class that can take any astropy.wcs.wcsapi.BaseLowLevelWCS
+    object and expose the high-level WCS API.
+
+allOf:
+  - type: object
+    properties:
+      wcs:
+        anyOf:
```

Comment:
It would be really good to have a way of saying "Any APE-14 compliant WCS". I am kind of tempted to allow any object inside these WCS wrapper classes as we can't possibly enumerate all of them?

Now I think about it #276 has the same problem with the changes to the SlicedLowLeveWCS schema, we should be allowing all the NDCube wrappers there as well.